float32 b_sp
float32 b_fb
float32 c_sp
float32 c_fb
float32 d_sp
float32 d_fb
float32 e_sp
float32 e_fb
float32 prop_sp
float32 prop_rps
